developer's code of conduct.md

```diff
@@ -24,17 +24,22 @@ To improve review quality, in PR about new feature/feature change/balance change
-      2) Thaumcraft lives and dies on the quality of the lore, while there are some allowances for data here and there, hard numbers are meant for the questbook, not the thaumonomicon. 
+      2) Thaumcraft lives and dies on the quality of the lore, while there are some allowances for data here and there, hard numbers are meant for the questbook, not the thaumonomicon.
       3) Create lore entries with the upmost seriousness, joke entries ***are*** allowed, however, this is meant for singular isolated items and chains, please do not create an entire research tab for trivial things or jokes.
       4) Intentionally misleading data is explicitly forbidden, for example, please do not add in a mention of "[one low low price of [2000 warp]](https://github.com/GTNewHorizons/thaumicinsurgence/commit/8e941936a5fe3fa0bee05a3f498f571859da3136)" within the text of a research entry that actually does contain warp, users will absolutely panic.
       5) Please refrain from adding any content directly to the vanilla Thaumcraft tabs, not only is this a direct request by Azanor in the terms of Thaumcraft addon creation, but likewise, it just causes issues with unlocking Kami. ([SuperTiC + Kami bug](https://github.com/GTNewHorizons/Super-TiC/pull/3))
       6) Please don't swear within research or tooltips, there are some allowances, such as certain usages of "damn/damned" but in general it comes off [crass and childish](https://github.com/GTNewHorizons/thaumicinsurgence/commit/8e941936a5fe3fa0bee05a3f498f571859da3136).
    3) Blood Magic
-      1) 
+      1)
    4) Botania
-      1) 
+      1)
    5) Witchery
        1) Don't, if you want to add Witchery content, please add it to Forbidden Magic instead. Witchery  exists as a singular ARR island compared to the archipelagos that make up the other magic mods, no vast fields of addons like Thaumcraft, no open sourced code like Botania and Blood Magic, if you'd like to see new Witchery content, please help us recreate the vital portions in new, untethered, code.
+9) Expedition on termination
+   1) There are some allowances on the expedition of termination of certain pack content, i.e. not having to wait a full development cycle (typically half a year) under these specific circumstances.
+      1) At least 1/4th of the GregTech New Horizons development team agree to the instataneous termination.
```

Comment:
from where does this 1/4 comes? and it's not the majority anyways...

Reply:
I would say it must be the majority

Reply:
Have y'all ever tried to get even 10 devs to agree on something?

Reply:
that's the point of this very repository so yes

Reply:
Genuinely count the number of devs on the entire team that are consistently active in the community. 

I've already closed this pr, because frankly I don't see this going anywhere at this current moment, but 1/4th of the overall total amount of devs (estimated to be like 13-15ish) is a more than reasonable amount.